applications/clouddemo/netcore/Dockerfile

```
@@ -19,15 +19,15 @@
 # under the License.
 #
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1
+FROM mcr.microsoft.com/dotnet/core/aspnet:6.0
```

```suggestion
FROM mcr.microsoft.com/dotnet/aspnet:6.0
```
See here: https://mcr.microsoft.com/en-us/product/dotnet/aspnet/about. It might be that the `\core\` path segment is supported even (not certain), but still we should go with the oficial tags. And it does read very weird to have `core` for .NET 6.